src/components/query/viewer/RecursiveWhereDisplay.vue

```diff
@@ -58,10 +68,12 @@
 
 <script setup lang="ts">
 import { isArrayHasLength } from "@/helpers/DataTypeCheckers";
-import { Where, Assignable, Bool } from "@/interfaces/AutoGen";
+import { Where, Assignable, Bool,Node } from "@/interfaces/AutoGen";
```

Comment:
Setup prettier for intellij and format on file save to avoid formatting issues like this.
https://www.jetbrains.com/help/idea/prettier.html#ws_prettier_configure

https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html